tests/run_all_tests.py

```diff
@@ -4,5 +4,8 @@
     loader = TestLoader()
     tests = loader.discover('.')
     testRunner = TextTestRunner()
-    testRunner.run(tests)
+    test_results = testRunner.run(tests)
+
+    if len(test_results.errors) > 0:
+        exit(1)
```

Comment:
Returning a non-zero exit code causes `python run_all_tests.py` to tell the OS that it "failed". This results in automated testing to fail properly when any errors are reported.